### ISSUE TITLE
Lost commit history?

### ISSUE THREAD

**Issue by tunnell** [SNEWS2/hop-plugin-snews#2]
This package seems like a copy-paste of the [package here](https://github.com/RiceAstroparticleLab/hop-SNalert-app), which means that we lost the commit history and can't see how previous development (or issue tracker) was done.  Can we either fork the other repository, or I can transfer ownership here if that is preferred?

**Comment by tunnell** [comment]
@myNameIsPatrick

**Comment by myNameIsPatrick** [comment]
Hi @tunnell, apologies for the late reply. The packages certainly look similar and that is mostly intentional. Both the snews plugin and the app repos were created using cookiecutter templates (https://github.com/scimma/hop-plugin-template, https://github.com/scimma/hop-app-template). The plugin template was developed to aid in creating a repo for plugins and was motivated by the SNEWS work, and a snapshot of the plugin codebase was migrated to this repo in this commit (https://github.com/RiceAstroparticleLab/hop-SNalert-app/commit/2d5641309b49e04ac41e9f025f634b031208c44d). It did unfortunately make it more difficult to track the commit history before this point in this repo (it is still captured in the original repo, however).

**Comment by bfc5288** [comment]
@tunnell To clarify a few points, this `hop-plugin-snews` package is similar but separate from the `snews` app itself (which is still in the [RiceAstroparticleLab repo](https://github.com/RiceAstroparticleLab/hop-SNalert-app)).

This `hop-plugin-snews` code doesn't provide the core SNEWS functionality (message storage, deciding, alerting, etc.) -- all of that functionality is still in the original [RiceAstroparticleLab repo](https://github.com/RiceAstroparticleLab/hop-SNalert-app)). This repo just provides the message formats that're used by that app (though as @myNameIsPatrick mentioned, the codebases have the same skeleton based on SCiMMA's cookiecutter templates).

This means that the core app (and development history) are still all contained in the original repository (and so a Zenodo reference could reasonably point there). Though it may be possible to consolidate the core code with the plugin within the SNEWS organization, if there's interest.

**Comment by tunnell** [comment]
Yeah, it might make sense to consolidate.  Thanks for explaining.  At very least, @yiyangxu1998 can you move the Rice repository here?

**Comment by skylerxu1998** [comment]
Yes give me a little time. I'll get back shortly.

**Comment by skylerxu1998** [comment]
@tunnell @bfc5288 We would want to transfer the repo right?

**Comment by bfc5288** [comment]
> @tunnell @bfc5288 We would want to transfer the repo right?

Yes, I would say so. It looks like there is a safe method to do this without losing the history of commits, pull requests, issues, etc. based on [the Github docs](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/transferring-a-repository).

**Comment by skylerxu1998** [comment]
Yeah just wanted to make sure tranferring is the way we want. Thanks

**Comment by skylerxu1998** [comment]
@tunnell The repo is linked to Zenodo, but I cannot create a release after trasferring the repo. I think it might need permission.
